ecmd-core/pyecmd/test_api.py

```diff
@@ -10,3 +10,6 @@
      #unit_id_string = unitIdToString(2)
      #clock_state = t.queryClockState("SOMECLOCK")
      t.relatedTargets("pu.c")
+     retval = t.queryFileLocation(ECMD_FILE_SCANDEF, "")
+     for loc in retval.fileLocations:
+         print loc.textFile, loc.hashFile, retval.version
```

Comment:
This print, in combo with what is loaded in ecmdDllStub.C, creates this at the end of a build:
`***** If you see python load errors this build of the python module is invalid ****
bogus/file.text bogus/file.hash v8`
That doesn't look great at the end of the build, I thought something had gone wrong.  I would suggest removing the print and just check the rc from the API call.  Throw a message if non-zero.

Reply:
I love how those lingering doubts at the back of my head always come back to bite me :) I'll remove the print, thanks!